### PR TITLE
Update readme for 2024

### PR DESCRIPTION
**Pull request by tsykoduk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,29 @@
...
 The best way to get started is to fork this repository and experiment with the examples.
+The examples as provided are designed for the use of PGO along with Crunchy Data's Postgres distribution, Crunchy Postgres, as Crunchy Postgres for Kubernetes.  For more information on the use of container images downloaded from the Crunchy Data Developer Portal or other third party sources, please see 'License and Terms' below.
 
-Each of the examples has its own README that guides you through the process of deploying it.
+### Help with the Examples
+
+* For general questions or community support, we welcome you to join our [community Discord](https://discord.gg/BnsMEeaPBV).
+* If you believe you have discovered a bug, please open an issue in the [PGO project](https://github.com/CrunchyData/postgres-operator).
+* You can find the full Crunchy Postgres for Kubernetes documentation [here](https://access.crunchydata.com/documentation/postgres-operator/v5/).
+* You can find out more information about PGO, the Postgres Operator from [Crunchy Data](https://www.crunchydata.com) at the [project page](https://github.com/CrunchyData/postgres-operator).
```
Comment (cbandy) [inline README.md:19]:
```suggestion
* You can find out more information about PGO, the Postgres Operator from [Crunchy Data](https://www.crunchydata.com), at the [project page](https://github.com/CrunchyData/postgres-operator).
```

File: README.md [hunk 2]
```diff
@@ -1,19 +1,29 @@
...
+
+### FAQs, License and Terms
 
-You can find the full [PGO documentation](https://access.crunchydata.com/documentation/postgres-operator/v5/) for the project here:
+For more information regarding PGO, the Postgres Operator project from Crunchy Data, and Crunchy Postgres for Kubernetes, please see the [frequently asked questions](https://access.crunchydata.com/documentation/postgres-operator/latest/faq).
 
-[https://access.crunchydata.com/documentation/postgres-operator/v5/](https://access.crunchydata.com/documentation/postgres-operator/v5/)
+For information regarding the software versions of the components included in and Kubernetes version compatibility, please see the [components and capabilities section of the Crunchy Postgres for Kubernetes documentation](https://access.crunchydata.com/documentation/postgres-operator/latest/references/components).
```
Comment (cbandy) [inline README.md:25]:
```suggestion
For information regarding the software versions of the components included and Kubernetes version compatibility, please see the [components and capabilities section of the Crunchy Postgres for Kubernetes documentation](https://access.crunchydata.com/documentation/postgres-operator/latest/references/components).
```

Comment (andrewlecuyer) [inline README.md:25]: "Components and Capabilities" should be "Components and _Compatibility_".

https://access.crunchydata.com/documentation/postgres-operator/latest/references/components

File: README.md [hunk 3]
```diff
@@ -1,19 +1,29 @@
...
 
-You can find out more information about [PGO](https://github.com/CrunchyData/postgres-operator), the [Postgres Operator](https://github.com/CrunchyData/postgres-operator) from [Crunchy Data](https://www.crunchydata.com) at the project page:
+The examples provided in this project repository are available subject to the [Apache 2.0](https://github.com/CrunchyData/postgres-operator-examples/blob/main/LICENSE.md) license with the PGO logo and branding assets covered by our [trademark guidelines](https://github.com/CrunchyData/postgres-operator/blob/master/docs/static/logos/TRADEMARKS.md).
```
Comment (cbandy) [inline README.md:27]: A dash here always directs to the main branch, whatever it is named.

```suggestion
The examples provided in this project repository are available subject to the [Apache 2.0](https://github.com/CrunchyData/postgres-operator-examples/blob/-/LICENSE.md) license with the PGO logo and branding assets covered by our [trademark guidelines](https://github.com/CrunchyData/postgres-operator/blob/-/docs/static/logos/TRADEMARKS.md).
```

**Review by andrewlecuyer — CHANGES_REQUESTED** [review]
Looks great, just one small change needed.

File: README.md [hunk 4]
```diff
@@ -1,19 +1,29 @@
...
+* You can find out more information about PGO, the Postgres Operator from [Crunchy Data](https://www.crunchydata.com), at the [project page](https://github.com/CrunchyData/postgres-operator).
+
+### FAQs, License and Terms
 
-You can find the full [PGO documentation](https://access.crunchydata.com/documentation/postgres-operator/v5/) for the project here:
+For more information regarding PGO, the Postgres Operator project from Crunchy Data, and Crunchy Postgres for Kubernetes, please see the [frequently asked questions](https://access.crunchydata.com/documentation/postgres-operator/latest/faq).
 
-[https://access.crunchydata.com/documentation/postgres-operator/v5/](https://access.crunchydata.com/documentation/postgres-operator/v5/)
+For information regarding the software versions of the components included and Kubernetes version compatibility, please see the [components and capabilities section of the Crunchy Postgres for Kubernetes documentation](https://access.crunchydata.com/documentation/postgres-operator/latest/references/components).
```
Comment (cbandy) [inline README.md:25]:
```suggestion
For information regarding the software versions of the components included and Kubernetes version compatibility, please see the [components and compatibility section of the Crunchy Postgres for Kubernetes documentation](https://access.crunchydata.com/documentation/postgres-operator/latest/references/components).
```

**Review by cbandy — APPROVED** [review]

**Review by andrewlecuyer — APPROVED** [review]

**Pull request merged**